agent/agent-tooling/src/main/java/com/microsoft/applicationinsights/agent/internal/diagnostics/DiagnosticsHelper.java

```diff
@@ -73,6 +77,12 @@ public static boolean isFunctionsRpIntegration() {
     return functionsRpIntegration;
   }
 
+  public static void lazyUpdateVmRpIntegration(MetadataInstanceResponse response) {
+    rpIntegrationChar = 'v';
```

Comment:
I think this is a bit confusing here, as DiagnosticsHelper is only for integrated rp attach, and we don't have that for VMs, maybe it would help to move `rpIntegrationChar` somewhere else?

Reply:
where do you recommend? i can move it

Reply:
move it to a new class call SdkVersionPrefixHelper.